.vscode/tasks.json

```diff
@@ -88,7 +88,7 @@
       "label": "test",
       "type": "shell",
       "options": {
-        "cwd": "${workspaceFolder}/packages/cadl"
+        "cwd": "${workspaceFolder}/packages/typespec"
```

Comment:
Hmm, this is broken before too, it should have been packages/compiler. Guessing nobdy uses this. I use test explorer or run in terminal.

Reply:
Post rename